File: model_dflowfm_blackbox/tests/simple_waal_salt_kalman/plot_simulation.m
 fprintf('Attention: results only useful after running Simulation.oda \n')

 file = '.\stochModel\work0\DFM_OUTPUT_simplewaal_salt\simplewaal_salt_his.nc';
 time_work = ncread(file,'time');
 time_work = time_work/(24*60*60);
 sal_work = ncread(file,'salinity');
 
 figure(1)
 plot(time_work,sal_work(1,:),'b')
 title('station01')
 xlabel('time (days)');
 ylabel('salinity');
 
 figure(2)
 plot(time_work,sal_work(2,:))
 title('station02')
 xlabel('time (days)');
 ylabel('salinity');
 
 figure(3)
 plot(time_work,sal_work(3,:))
 title('station03')
 xlabel('time (days)');
 ylabel('salinity');
 
 

File: model_dflowfm_blackbox/tests/estuary_kalman_bc-file/plot_simulation.py
#! /usr/bin/python
# -*- coding: utf-8 -*-
"""
    Plot results after running Simulation.oda
"""
import matplotlib.pyplot as plt
from netCDF4 import Dataset

from datetime import datetime, timedelta

print("WARNING: results only useful after running Simulation.oda\n")

file = './stochModel/work0/dflowfm/DFM_OUTPUT_estuary/estuary_his.nc'
data = Dataset(file, 'r')

time = data.variables['time'][:]
wl = data.variables['waterlevel'][:]

fig1 = plt.figure()
plt.plot(wl[:, 0])
plt.title('Obs01')
plt.xlabel('time')
plt.legend(['prediction'])

fig2 = plt.figure()
plt.plot(wl[:, 1])
plt.title('Obs02')
plt.xlabel('time')
plt.legend(['prediction'])

fig3 = plt.figure()
plt.plot(wl[:, 2])
plt.title('Obs03')
plt.xlabel('time')
plt.legend(['prediction'])

plt.show()
